src/test/cli/ceph-authtool/simple.t

```diff
@@ -22,4 +22,6 @@
     --cap SUBSYSTEM CAPABILITY    will set the capability for given subsystem
     --caps CAPSFILE               will set all of capabilities associated with a
                                   given key, for all subsystems
+    --mode MODE                   will set the desired file mode to the keyring
+                                  e.g: '0644', defaults to '0600'
   [1]
```

Comment:
Thanks for updating the manpage test. Jenkins is supposed to catch that `doc/man/8/ceph-authtool.rst` also needs an update here.

Reply:
repushed thanks @ktdreyer